assets/src/components/child/Child.js

```diff
@@ -0,0 +1,45 @@
+import zdClient from "./../../libs/ZDClient.js";
+import childTemplate from "./ChildTemplate.js";
+import { splitIdentities } from "./../../libs/Helper.js";
+
+const Child = {
+  template: childTemplate,
+  props: {
+    dictionary:Object,
```

Comment:
you are importing dictionary for all components while if you install as vue plugin we can install it once and use from everywhere